kubernetes.go

```diff
@@ -211,6 +211,12 @@ func (c *KubernetesClient) getServices() ([]ServiceInformation, error) {
 				continue
 			}
 
+			found := InEphemeralPortsRange(s.Spec.Ports)
+			if found {
```

Comment:
As you are not using `found` later and is a simple condition you could put it directly in the if (`if InEphemeralPortsRange(s.Spec.Ports) {...}`)